cmd/fyne/internal/mobile/manifest.go

```diff
@@ -53,14 +53,16 @@ type manifestTmplData struct {
 	Name        string
 	Debug       bool
 	LibName     string
+	Version     string
+	Build       int
 }
 
 var manifestTmpl = template.Must(template.New("manifest").Parse(`
```

Comment:
Should this perhaps be moved out to a separate file in the internal template package?

Reply:
These files are still a fork of the gomobile build tools, howeover loosely. Moving the content around will make it very difficult to apply any security / feature patches from upstream.
At some point we may choose to do that, but I don't think just now is that time.

Reply:
Good point. Let's no do that now.

Reply:
So is this approved as that change is not needed?